src/discordtipbot/tipbot.cr

```diff
@@ -70,6 +66,25 @@ class TipBot
     true
   end
 
+  def process_pending_withdrawals
+    tuple = {id: Int32, from_id: Int64, address: String, amount: BigDecimal}
```

Comment:
I might recommend a `record` instead.

Reply:
Mind elaborating? calling what I'm calling `tuple` `record`?

Reply:
After thinking about it, I think that's what you meant :smiley: